jdaviz/configs/rampviz/plugins/ramp_extraction/ramp_extraction.py

```diff
@@ -146,6 +146,9 @@ def _on_subset_update(self, msg={}):
             # if called before fully initialized
             return
 
+        if not hasattr(self.app._jdaviz_helper, '_default_integration_viewer_reference_name'):
+            return
+
```

Comment:
I think this will prevent ramp functionality from running in deconfigged entirely.  Instead can we use `self.app.get_viewers_of_cls(RampvizProfileView)[0]`?  (in `self.integration_viewer` and then removing these if-statements)

Reply:
done